exist-core/src/main/java/org/exist/xquery/pragmas/Optimize.java

```diff
@@ -39,15 +39,17 @@
 import java.util.Iterator;
 import java.util.List;
 
-public class Optimize extends Pragma {
+import static java.lang.System.arraycopy;
```

Comment:
is this needed?

Reply:
arraycopy is used once

Reply:
Typically the Java convention is not to statically import this, the convention is to instead write `System.arrayCopy(...)` at the call-site.